apps/server/src/modules/deletion-console/deletion-execution.console.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { createMock, DeepMocked } from '@golevelup/ts-jest';
 import { ConsoleWriterService } from '@infra/console';
-import { DeletionExecutionUc } from '../uc';
+import { DeletionExecutionUc } from '../deletion/uc';
```

Comment:
```suggestion
import { DeletionExecutionUc } from '@modules/deletion';
```
!!! Circular Dependency